operations/reservations.md

```diff
@@ -289,222 +292,292 @@ Returns all reservations specified by any identifier, customer or other filter.
-| `Extent` | [Reservation extent](#reservation-extent) | required | Extent of data to be returned. E.g. it is possible to specify that together with the reservations, customers, groups and rates should be also returned. |
+| `ChannelNumbers` | array of string | optional, max 1000 items | Set of numbers or references used by the Channel (i.e. OTA, GDS, CRS, etc.) in case the reservation group originates there, e.g. Booking.com confirmation numbers. |
+| `Numbers` | array of string | optional, max 1000 items | Confirmation numbers of [Reservations](reservations.md#reservation-ver-2017-04-12). |
+| `StartUtc` | string | optional | Start of the interval in UTC timezone in ISO 8601 format. Required when used in conjunction with the TimeFilter or States search parameter. |
+| `EndUtc` | string | optional | End of the interval in UTC timezone in ISO 8601 format. Required when used in conjunction with the TimeFilter or States search parameter. |
+| `TimeFilter` | [Reservation time filter](reservations.md#reservation-time-filter) | required | Time filter of the interval. If not specified, reservations Colliding with the interval are returned. |
 | `Currency` | string | optional | ISO-4217 code of the [Currency](currencies.md#currency) the item costs should be converted to. |
-| `Limitation` | [Limitation](../guidelines/pagination.md#limitation) | required | Limitation on the quantity of reservation data returned. |
+| `States` | array of [Service order state (ver 2017-04-12)](reservations.md#service-order-state-ver-2017-04-12) | optional | States the reservations should be in. If not specified, reservations in Confirmed, Started or Processed states or reservations specified by ReservationIds regardless of state are returned. |
+| `Extent` | [Reservation extent](reservations.md#reservation-extent) | required | Extent of data to be returned. E.g. it is possible to specify that together with the reservations, customers, groups and rates should be also returned. |
+| `Limitation` | [Limitation](../guidelines/pagination.md#limitation) | required | Limitation on the quantity of data returned. |
+| ~~`ServiceId`~~ | ~~string~~ | ~~optional~~ | **Deprecated!** |
 
 #### Reservation time filter
 
-* `Colliding` - reservations whose intervals collide with the specified interval.
-* `Created` - reservations created within the specified interval.
-* `Updated` - reservations updated within the specified interval.
-* `Start`- reservations starting \(= arriving\) within the specified interval.
-* `End` - reservations ending \(= departing\) within the specified interval.
-* `Overlapping` - reservations whose intervals contain the specified interval.
-* `Canceled` - reservations canceled within the specified interval.
+* `Colliding`
+* `Created`
+* `Updated`
+* `Start`
+* `End`
+* `Overlapping`
+* `Canceled`
+* `Cancelled`
```

Comment:
Descriptions are gone and the "Cancelled" value has appeared. I suppose we don't support enum value deprecation in generator?